addon/components/md-select.js

```diff
@@ -7,6 +7,9 @@ export default MaterializeInputField.extend({
   classNames: ['md-select'],
   optionLabelPath: 'content',
   optionValuePath: 'content',
+  contentChanged: Ember.observer('content.[]', function() {
+    Ember.run.once(this, '_setupSelect');
+  }),
```

Comment:
Also since this is being scheduled in the run loop, you'll need to add some guards in `_setupSelect` in case by the time the run loop queue flushes, we do no work if the component is destroyed or is being destroyed.

```js
if (!this.isDestroying && !this.isDestroyed) {
  // do some stuff
}
```
Failure to do this may result in errors like "calling set on destroyed object". I'm sure you've seen some of those :)